src/components/Navigation.vue

```diff
@@ -100,15 +108,21 @@ export default {
 	computed: {
 		menu() {
 			return this.$store.getters.accounts.map((account) => {
-				const folders = this.$store.getters
-					.getFolders(account.id)
-					.filter((folder) => !account.collapsed || SHOW_COLLAPSED.indexOf(folder.specialRole) !== -1)
-				//.map(folderToEntry)
+				const folders = this.$store.getters.getFolders(account.id)
+				const nonSpecialRoleFolders = folders.filter(
+					(folder) => SHOW_COLLAPSED.indexOf(folder.specialRole) === -1
+				)
+				const isCollapsible = nonSpecialRoleFolders.length > 1
+				const foldersToShow = folders.filter(
```

Comment:
is this filter necessary? The template does that already in the `v-show`, right?

Reply:
I think you're right. It got introduced in 47ff8d0 in the template and I cannot see when this would ever have an effect. Why is it being used on the `show` attribute? Is there some kind of animation planned?

Reply:
> Is there some kind of animation planned?

Ideally yes, the "Expand/Collapse" control should move down to the bottom in a quick animation, revealing the folders. We have `--animation-quick:100ms;` or `--animation-slow:300ms;` for that, and the -slow is probably the fitting one here.

Also :wave: @xMartin :D

Reply:
Yeah, but let's do the animation in a separate PR :)

Reply:
@jancborchardt 👋

Reply:
@ChristophWurst I cleaned it up. The starred inbox is always shown anyway if it's rendered so I removed the condition there. Actually there's a bug that the inbox is always rendered twice. Not sure how it's supposed to work but I see it as a seperate problem.

Reply:
> ctually there's a bug that the inbox is always rendered twice. Not sure how it's supposed to work but I see it as a seperate problem.

Fixed via https://github.com/nextcloud/mail/pull/2803 :)

Reply:
>  The starred inbox is always shown anyway if it's rendered so I removed the condition there.

Good catch. That check is indeed useless :)